app/models/request.rb

```diff
@@ -10,7 +10,7 @@ class Request < ApplicationRecord
   validate :validate_pending_status, on: :update
 
   def is_requested_by?(user)
-    raise(StandardError, 'Request not reachable') && (return false) unless requester == user
+    raise(StandardError, 'This is not your reQuest') && (return false) unless requester == user
```

Comment:
Won't raising the error override the remaining execution and make the return redundant?

Reply:
Yes, that worked.